### ISSUE TITLE
Sending Commands to H02 Device with RESTAPI

### ISSUE THREAD

**Issue by WalrusHat** [vitalidze/traccar-web#746]
I have an H02 provisioned device and I'm trying to send the stop engine command via the API. Is this possible?

**Comment by vitalidze** [comment]
Looks like support for 'engine stop' command is implemented in the backend: https://github.com/tananaev/traccar/blob/master/src/org/traccar/protocol/H02ProtocolEncoder.java#L51

It should be possible to send commands through API. You can find some info in #571 #612

**Comment by WalrusHat** [comment]
Thanks Vitalidze! I'll look into it. Seriously you are more available for support then my church! Thank you for all the help.

**Comment by tghilani** [comment]
Hi all,
I have the same question for PT502? I got this error:
"{\"reason\":\"java.lang.RuntimeException: Command engineStop is not supported in protocol pt502\",\"success\":false}"

I think it is not support?

Thanks for your answer

**Comment by vitalidze** [comment]
@tghilani that's correct: it means command is not yet implemented on the traccar backend.

**Comment by tghilani** [comment]
Thanks Vitalize you think it is easy to add it, I am thinking to work on the protocol and impllement it?. (Long time did not code any line in java ;-).

**Comment by vitalidze** [comment]
@tghilani it should not be very hard if you have protocol documentation.

**Comment by rafarngt** [comment]
Hi, when I send the command on and off of the device (Suntech 340lc) I get the following error: {\ "reason \": \ "java.lang.ClassNotFoundException: org.traccar.http.JsonConverter \", \ "success \ ": False}. Could help me with this. I have the 3.9 version of traccar.

**Comment by vitalidze** [comment]
@rafarngt please don't resurrect old threads. You should create your own instead. In your case I think you may be using old version of Traccar Web UI Mod.

**Comment by rafarngt** [comment]
I understand, finally, where could you get the new version of the user interface for v3.9?

**Comment by vitalidze** [comment]
@rafarngt you can find a link in usual place here: http://traccar.litvak.su/installation.html
